unicoremc/views.py

```diff
@@ -259,15 +261,17 @@ class AppEventSourceView(ProjectViewMixin, View):
 
     def get(self, request, project_id, task_id, path):
         project = get_object_or_404(Project, pk=project_id)
```

Comment:
Can you use `self.get_queryset()` here like in `AppLogView` [here](https://github.com/universalcore/unicore-mc/blob/feature/issue-120-fix-redirect/unicoremc/views.py#L249)? Should I open an issue? It limits projects to ones the user has permission to access. Sorry, I didn't review the PR that made that change.